client/pages/home.jsx

```diff
@@ -6,39 +6,64 @@ const App = () => {
   const [searchText, setSearchText] = useState('');
   const [searchCity, setSearchCity] = useState('');
   const [imageLinks, setImageLinks] = useState([]);
+  const [title, setTitle] = useState('Results');
+  const [imageFlag, setImageFlag] = useState(false);
 
   useEffect(() => {
-    venues.forEach((venue, i) => {
+    if (imageFlag) { getImages(); }
+    if (localStorage.getItem('favorites')) { favorites(); }
+  }, [venues]);
 
-      if (i > 1) return null;
+  const getImages = e => {
+    if (!imageFlag) { return; }
+    venues.forEach((venue, i) => {
 
       fetch(`/api/venue/image/${venue.id}`)
         .then(data => data.json())
         .then(res => {
           const newImageLinks = imageLinks.concat({ id: venue.id, url: res.imgUrl });
+
           if (res.imgUrl) { venues[i].imageUrl = res.imgUrl; } else {
-            venues[i].imageUrl = 'https://cdn.stocksnap.io/img-thumbs/960w/sliced-homemade_5BADCUBZS9.jpg';
+
+            venues[i].imageUrl = 'http://localhost:3001/images/noimage.jpeg';
```

Comment:
you can just put `/images/noimage.jpeg` instead of the whole URL. Otherwise, this will break when you deploy your app.